.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,49 @@
+---
+name: Pull Request
+about: Creating a PR
+
+---
+
+<!--
+        Hi there!
+        Thanks for taking the time and putting the effort into making fabric better! 💖
+        Take a look at ./CONTRIBUTING.md for crucial instructions regarding local setup, testing etc.
+        https://github.com/fabricjs/fabric.js/blob/master/CONTRIBUTING.md
+
+        Adding tests that verify your fix and safegurad it from unwanted loss and changes is a MUST.
+
+        PRing is not always simple, don't hestitate to ask for help (beware of 'em gotchas 😓).
+        We appreciate your effort and would like the process to be productive and enjoyable.
+        A strong community means a stong and better product for everyone.
+-->
+
+
+<!--        
+        📣 IMPORTANT NOTICE - PR LOCKDOWN 🔒    04/2022
+        We are excited to announce that fabric is migrating to modern typescript/javascript 🤩.
+        This means we will ⛔ not be accepting any PRs out of scope with the migration.
+        We understand this might be annoying but wasted work is ever more so.
+        The migration will be extreme on the source code so PRs from before will probably become stale to the point of death after the migration.
+        It hurts us the throw away good work, effort and time put into fabric so please stay patient.
+        You are welcome to join the migration effort 🔨
+        https://github.com/fabricjs/fabric.js/issues/7596
```

Comment:
We can also say that if they have a small fix they can still PRO it to the v5.x branch. If is important or not already solved in the migration, can be ported after.

Reply:
DONE [304dd2f](https://github.com/fabricjs/fabric.js/pull/7857/commits/304dd2f49d3ca82d150e0ec8f2d7cba4ccd31b94)